backend/handlers/handlers.go

```diff
@@ -1438,6 +1439,32 @@ func (handlers *Handlers) getExchangeMoonpayBuyInfo(r *http.Request) (interface{
 	return resp, nil
 }
 
+func (handlers *Handlers) getBtcDirectInfo(r *http.Request) (interface{}, error) {
```

Comment:
Please, rename to `getExchangeBtcDirectInfo` to follow the existing convention :innocent: 
Also, returning `(interface{}, error)` is kind of deprecated, as it is not correctly handled in the frontend. Better to just return `interface{}` and use the `success` param and an error message